include/mlboard_logger.hpp

```diff
@@ -0,0 +1,200 @@
+/**
+ * @file mlboard_logger.hpp
+ * @author Jeffin Sam
+ *
+ * Implementation of the mlbaord logger callback function.
+ */
+#ifndef ENSMALLEN_CALLBACKS_MLBOARD_LOGGER_HPP
+#define ENSMALLEN_CALLBACKS_MLBOARD_LOGGER_HPP
+
+#include <ensmallen.hpp>
+#include <mlboard/core.hpp>
+#include <mlboard/mlboard.hpp>
+
+namespace ens {
+
+/**
+ * Mlboard Logger function, based on the EndEpoch callback function.
+ */
+class MlboardLogger
+{
+ public:
+  /**
+   *
+   * @param output Filewriter object to log the metrics.
+   * @param epochCount Interval of epochs youw want to log your data.
+   * @param accTag Tag to use for accuracy.
+   * @param lossTag Tag to use for loss
+   */
+  MlboardLogger(
+      mlboard::FileWriter& output,
+      int epochCount = 1,
+      std::string accTag = "accuracy",
+      std::string lossTag = "loss") : callbackUsed(false),
```

Comment:
```suggestion
      std::string lossTag = "loss") : callbackUsed(false),
```
      I think this is not correct style.